Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -339,9 +339,13 @@ theorem type_subrel_lt (o : Ordinal.{u}) :
   rintro ⟨α, r, wo⟩; apply Quotient.sound
   constructor; refine ((RelIso.preimage Equiv.ulift r).trans (enum r).symm).symm
 
-theorem mk_initialSeg (o : Ordinal.{u}) :
-    #{ o' : Ordinal | o' < o } = Cardinal.lift.{u + 1} o.card := by
+theorem mk_Iio_ordinal (o : Ordinal.{u}) :
+    #(Iio o) = Cardinal.lift.{u + 1} o.card := by
   rw [lift_card, ← type_subrel_lt, card_type]
+  rfl
+
+@[deprecated mk_Iio_ordinal (since := "2024-09-18")]
+alias mk_initialSeg := mk_Iio_ordinal
```

Comment:
Instead of using an `alias`, you should keep the theorem on the original form as written. That minimizes the number of downstream problems.